ui/Modal/Org/ConfigureEns/BlockTimer.svelte

```diff
@@ -22,6 +22,10 @@
   let done: boolean = false;
 
   const onBlock = (currentBlock: number) => {
+    if (!startBlock) {
+      startBlock = currentBlock;
```

Comment:
Instead of doing this, we should save the transaction hash of the commitment transaction, fetch the receipt for that tx and extract the block number from there.

It would probably be best to do this right after we call `restoreCommitment`. Then, `startBlock` would not need to be `undefined` and we don’t need to handle that case in the UI.

Reply:
5df5994c93784b480fc1866be125de317b11e6f8